packages/fuels-core/src/types/raw_slice.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{Parameterize, Token, Tokenizable};
+use fuels_types::errors::Error;
+use fuels_types::param_types::ParamType;
+use std::fmt::{Debug, Display, Formatter};
+
+// To be used when interacting with contracts which have strings in their ABI.
```

Comment:
I'm not sure the explanation about strings and their length is warranted here.

I'd stop at: 
`RawSlice` is a mapping of the contract type `raw_slice` -- currently the only way of returning dynamically sized data from a contract.

Reply:
This is a leftover from a copy-paste haha